styleguide/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  preset: 'ts-jest/presets/js-with-ts',
```

Comment:
Seems like this would work fine, since we don't have any js in styleguide
```suggestion
  preset: 'ts-jest',
```